sparse/_common.py

```diff
@@ -33,6 +36,26 @@
 )
 
 
+@numba.njit
+def nan_check(*args):
+    for i in literal_unroll(args):
+        if np.isnan(np.min(np.asarray(i))):
+            return True
+    return False
+
+
+def check_class_nan(test):
+    from ._compressed import GCXS
+    from ._coo import COO
+
+    if isinstance(test, GCXS) or isinstance(test, COO):
```

Comment:
```suggestion
    if isinstance(test, (GCXS, COO)):
```

This works just fine, but can be cleaned up a bit.

Reply:
Sir this is final benchmark.
```
All benchmarks:

       before           after         ratio
     [e036c254]       [4053ce4b]
     <master~4>       <master>  
      2.78±0.08ms       2.85±0.1ms     1.03  benchmark_coo.ElemwiseBroadcastingSuite.time_add
          361±2μs         377±20μs     1.04  benchmark_coo.ElemwiseBroadcastingSuite.time_mul
      3.42±0.03ms       3.55±0.1ms     1.04  benchmark_coo.ElemwiseSuite.time_add
         999±10μs      1.02±0.04ms     1.02  benchmark_coo.ElemwiseSuite.time_mul
          521±4μs          551±4μs     1.06  benchmark_coo.IndexingSuite.time_index_fancy
       26.1±0.1μs       27.5±0.3μs     1.05  benchmark_coo.IndexingSuite.time_index_scalar
          117±2μs          120±3μs     1.02  benchmark_coo.IndexingSuite.time_index_slice
         283±10μs          277±9μs     0.98  benchmark_coo.IndexingSuite.time_index_slice2
         299±10μs          295±5μs     0.99  benchmark_coo.IndexingSuite.time_index_slice3
         340±10μs         348±10μs     1.02  benchmark_coo.MatrixMultiplySuite.time_matmul
      4.72±0.04ms       4.86±0.2ms     1.03  benchmark_gcxs.ElemwiseBroadcastingSuite.time_add
      1.16±0.03ms      1.19±0.04ms     1.03  benchmark_gcxs.ElemwiseBroadcastingSuite.time_mul
       6.95±0.2ms       6.88±0.2ms     0.99  benchmark_gcxs.ElemwiseSuite.time_add
      3.14±0.04ms       3.18±0.1ms     1.01  benchmark_gcxs.ElemwiseSuite.time_mul
+     1.57±0.02ms       1.77±0.3ms     1.13  benchmark_gcxs.IndexingSuite.time_index_fancy
          189±1μs          201±5μs     1.06  benchmark_gcxs.IndexingSuite.time_index_scalar
      1.56±0.03ms      1.63±0.01ms     1.04  benchmark_gcxs.IndexingSuite.time_index_slice
      1.03±0.01ms      1.07±0.02ms     1.04  benchmark_gcxs.IndexingSuite.time_index_slice2
         790±10μs         834±90μs     1.06  benchmark_gcxs.IndexingSuite.time_index_slice3
          342±3μs          378±6μs    ~1.11  benchmark_gcxs.MatrixMultiplySuite.time_matmul
          165±2ms        174±0.9ms     1.05  benchmark_tensordot.TensordotSuiteDenseSparse.time_dense
          165±2ms        176±0.8ms     1.06  benchmark_tensordot.TensordotSuiteDenseSparse.time_sparse
          134±2ms          144±2ms     1.07  benchmark_tensordot.TensordotSuiteSparseDense.time_dense
          134±2ms          146±2ms     1.09  benchmark_tensordot.TensordotSuiteSparseDense.time_sparse
         88.9±2ms         93.6±2ms     1.05  benchmark_tensordot.TensordotSuiteSparseSparse.time_dense
         88.3±2ms         92.2±2ms     1.04  benchmark_tensordot.TensordotSuiteSparseSparse.time_sparse

```